src/target/target_kind.cc

```diff
@@ -334,6 +334,7 @@ TVM_REGISTER_TARGET_KIND("cuda", kDLCUDA)
     .add_attr_option<Integer>("max_threads_per_block")
     .add_attr_option<Integer>("thread_warp_size", Integer(32))
     .add_attr_option<Integer>("registers_per_block")
+    .add_attr_option<Integer>("l2_cache_size_bytes")
```

Comment:
Nit, naming convention. Previous attributes like `max_shared_memory_per_block` is also using bytes, what about we stick to the same style and use `l2_cache_size`? Please feel free to add a comment to clarify it's in bytes though.

Reply:
Yeah I agree with that. It's only I am following the previously defined(but not implemented attribute) attribute in target, see https://github.com/apache/tvm/blob/main/python/tvm/target/target.py#L272

And the same attribute is defined and implemented for devices. See https://github.com/apache/tvm/blob/main/python/tvm/_ffi/runtime_ctypes.py#L492

The difference is that for targets, we will hardcoded this attibute (b/c we may build a model on devices different from the provided target), and for devices, we will automatically detect it from real devices.

Reply:
See #15332

Reply:
Hi, as I mentioend in the beginning of this commen, I was just nit picking and I don't have strong opinion if you prefer to keep it. On the other hand, I don't think any naming for this would impact your cross compilation usage, just make sure you have consistent naming with Zihao's PR should be good.

Reply:
I see, okay